### DISCUSSION TITLE
Desktop first approach

### DISCUSSION THREAD

**Discussion by IndranilBhattacharya** [tailwindlabs/tailwindcss#8064] [Ideas]
For responsive designs, we only have @media features with min-width condition.
Is there any plan to bring desktop-first approach? with max-width?

**Comment by CFarhad** [comment]
Hi 
It is exist by default. Check this [link](https://tailwindcss.com/docs/screens#max-width-breakpoints)

**Reply by IndranilBhattacharya** [reply]
Its not default, but the configuration works!
Thanks.
